docs/_posts/2017-09-12-520-release.md

```diff
@@ -27,6 +27,8 @@ I’m excited to bring you the news of Open GEE’s 5.2.0 Release! We’ve been
 * CentOS 7.x
 * Ubuntu 14.04 LTS and 16.04 LTS
 
+To download this release build see the [release page](https://github.com/google/earthenterprise/releases/tag/5.2.0-3.final). The full release notes can also be found [here](http://www.opengee.org/geedocs/answer/7160000.html).
```

Comment:
Minor: I would take out the word 'build' - i.e. "To download this release see the..."